scripts/build/config.js

```diff
@@ -426,11 +426,21 @@ const nonPluginUniversalFiles = [
     umdVariableName: "prettier",
     replaceModule: [
       {
-        module: require.resolve("@babel/highlight"),
+        module: require.resolve("@babel/highlight", {
+          paths: [require.resolve("@babel/code-frame")],
+        }),
         path: path.join(dirname, "./shims/babel-highlight.js"),
       },
       {
-        module: createRequire(require.resolve("vnopts")).resolve("chalk"),
+        module: require.resolve("chalk", {
+          paths: [require.resolve("@babel/code-frame")]
+        }),
+        path: path.join(dirname, "./shims/chalk.js"),
+      },
```

Comment:
This change (and the `getChalk` removal in `shims/babel-highlight.js`) are because of https://github.com/babel/babel/pull/15812.

You will need to do the same changes when we release the next Babel 7 version, as `@babel/code-frame` now has a `chalk` dependency.

Reply:
[`de94bcc` (#15236)](https://github.com/prettier/prettier/pull/15236/commits/de94bccca752f3e2db07ecd6c470735f9dd8590a)